### PR TITLE
London class 7 - Daniel Ghanbari - Github - week 1

### PR DESCRIPTION
**Pull request by DanGhanbari**
most of the question asweres were modified as they were wrong.

# Your Details

- Your Name: Daniel Ghanbari
- Your City: London
- Your Slack Name: London-class7

# Homework Details

- Module: Git
- Week: 1

### CODE REVIEW

**Review by nPelvin — COMMENTED** [review]
Hi Daniel, 

I have reviewed your changes and it all looks correct to me. 

Well done! 

Nigel

File: homework.md [hunk 1]
```diff
@@ -110,4 +108,4 @@ To record what changes you made to the code
 
 Where can I look to find help with Git?
 
-The Git Cheatsheet!
+Slack chanels!
```
Comment (meganlouiseh) [inline homework.md:111]: This is definitely right (Slack channels are useful) but I think the Git cheatsheet is a correct answer too.

**Review by meganlouiseh — COMMENTED** [review]
I have finished my review :)

File: homework.md (hunk 1 above)
Comment (DanGhanbari) [reply homework.md:111]: Thanks for your comment, you are right. I will add that to the answer.

**Pull request closed without merging**